docs/fedora-releases-guide.md

```diff
@@ -221,7 +221,17 @@ functionality. This means that Packit doesn't need to be set up in the
 upstream project: everything is configured in Fedora dist-git. So when a new
 upstream release happens and
 [release-monitoring.org](https://release-monitoring.org/) detects it, you'll
-get dist-git pull requests with it automatically. If you want to restrict what releases with corresponding tags Packit should react on, 
+get dist-git pull requests with it automatically. 
+
+The [`pull_from_upstream` job](/docs/configuration/downstream/pull_from_upstream) will close Bugzillas opened by 
+the Upstream Release Monitoring automatically on your behalf through the Bodhi updates. 
+Packit will add the Bugzillas numbers to the changelog in this form `- Resolves rhbz#XYZ`
+and [Bodhi will close them](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html#associate-bugs-to-automatic-updates). 
```

Comment:
could you please expand it and explain that:
- Bodhi does the autoclosing for rawhide
- Packit mirrors this behaviour for the updates created by Packit